install/overlays/kubernetes/README.md

```diff
@@ -0,0 +1,50 @@
+# Kubernetes overlay
+
+## Pre-requise
+
+This is an overlay intended for Minkube with the following configuration:
+* Cluster-admin privileges are required
+* Namespace is `default`
+* Operator id is `camel-k`
+* An available registry
+
+## Usage
+
+The following env variable are expected
+
+
+To run from local folder :
+```sh
+kubectl kustomize kustomize/overlays/kubernetes | kubectl create -f -
+```
+
+To run from remote github repository:
+```sh
+kubectl kustomize https://github.com/apache/camel-k/kustomize/overlays/kubernetes | kubectl create -f -
```

Comment:
With all examples, we should specify the ref in order to point to the latest released version. We need to update the release script in order to update the documentation when we release (could be a follow up issue though).